packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -123,7 +123,15 @@ export default {
             const { issues, isFulfillable } = responseData;
 
             // Can now log these errors inside the map if necessary
-            const detailedIssues = issues.map(issue => ({ ...checkoutIssues[issue.code], ...issue }));
+            const detailedIssues = issues.map(issue => {
+                const checkoutIssue = checkoutIssues[issue.code];
+
+                if (checkoutIssue) {
+                    return { ...checkoutIssue, ...issue };
+                }
+
+                return { code: DEFAULT_CHECKOUT_ISSUE, shouldShowInDialog: true };
```

Comment:
It's a subjective preference, but as for me it's a bit more readable if we do either
```
return checkoutIssue ? 
    { ...checkoutIssue, ...issue } : 
    { code: DEFAULT_CHECKOUT_ISSUE, shouldShowInDialog: true };
```
or
```
if (checkoutIssue) {
    return { ...checkoutIssue, ...issue };
} else {
    return { code: DEFAULT_CHECKOUT_ISSUE, shouldShowInDialog: true };
}
```

Reply:
To be honest, I think I did it this way because of eslint's pedantry!